src/sections/chat/ChatSidebarSection.tsx

```diff
@@ -211,6 +218,51 @@ const ChatSidebarSection = ({ showCreateGroupModal, setSelectedChatId }) => {
           />
         )}
 
+        {activeTab == 3 && (
```

Comment:
Ideally we should not have this number system but a complete tab name with proper types. Could you please add a todo comment in the codebase to fix it in future.

Reply:
done